galaxy/api/githubapi.py

```diff
@@ -58,7 +59,8 @@ def get_client(self):
                 "User does not have a GitHub OAuth token"
             )
         try:
-            client = Github(gh_token.token)
+            client = Github(base_url=settings.GITHUB_SERVER,
```

Comment:
To avoid confusion with `SOCIALACCOUNT_PROVIDERS['github'].GITHUB_URL` would it be better to indicate that this  url is related to API explicitly (e.g. `GITHUB_API_URL`)

Reply:
Understand, indeed it would make sense, though this would require to validate if `GITHUB_SERVER` is not used in other parts of the code. 
On Github Enterprise topic, I'm going to look into https://python-social-auth.readthedocs.io/en/latest/backends/github_enterprise.html and see if it would cover some of the changes I've included in this PR. Specifically, API is one of the required parameters.

Reply:
Yes, it would make sense. Though I'm not sure if `GITHUB_SERVER` parameter isn't used in other parts of the code.

Reply:
I've looked how it's done in **django-allauth** library (see https://github.com/pennersr/django-allauth/blob/master/allauth/socialaccount/providers/github/views.py ) and it looks like `GITHUB_SERVER` parameter in settings.py is not used at all. I implemented similar way in `galaxy\common\github.py`.